cdap-ui/app/cdap/components/PipelineConfigurations/Store/ActionCreator.js

```diff
@@ -45,36 +45,23 @@ import { CLOUD } from 'services/global-constants';
 // Filter certain preferences from being shown in the run time arguments
 // They are being represented in other places (like selected compute profile).
 const getFilteredRuntimeArgs = (runtimeArgs) => {
-  const RUNTIME_ARGS_TO_SKIP_DURING_DISPLAY = [
-    CLOUD.PROFILE_NAME_PREFERENCE_PROPERTY,
-    CLOUD.PROFILE_PROPERTIES_PREFERENCE,
-    'logical.start.time',
-  ];
   let { resolvedMacros } = PipelineConfigurationsStore.getState();
   let modifiedRuntimeArgs = {};
   let pairs = [...runtimeArgs.pairs];
-  const skipIfProfilePropMatch = (prop) => {
-    let isMatch = RUNTIME_ARGS_TO_SKIP_DURING_DISPLAY.filter(
-      (skipProp) => prop.indexOf(skipProp) !== -1
-    );
-    return isMatch.length ? true : false;
-  };
-  pairs = pairs
-    .filter((pair) => !skipIfProfilePropMatch(pair.key))
-    .map((pair) => {
-      if (pair.key in resolvedMacros) {
-        return {
-          notDeletable: true,
-          provided: pair.provided || false,
-          ...pair,
-        };
-      }
+  pairs = pairs.map((pair) => {
+    if (pair.key in resolvedMacros) {
       return {
+        notDeletable: true,
+        provided: pair.provided || false,
```

Comment:
Now that we have removed the checkboxes do we still need the provided?

Reply:
fixed